spec/resty/concurrent/event_spec.lua

```diff
@@ -0,0 +1,69 @@
+local Event = require('resty.concurrent.event')
+
+local yield = function () ngx.sleep(0) end
+
+describe('Event', function()
+    local threads = {}
+
+    local spawn = function(fun) -- helper function to track all threads and kill them between tests
+        local thread = ngx.thread.spawn(assert(fun))
+        table.insert(threads, thread)
+        return thread
+    end
+
+    after_each(function()
+        for i,thread in ipairs(threads) do
+            ngx.thread.kill(thread)
+            threads[i] = nil
+        end
+    end)
+
+    describe('can be waited by multiple threads', function()
+        it('notifies all threads', function()
```

Comment:
I think this title is not descriptive. What about `notifies the main thread` ?

Reply:
Yep, copy pasta description.